Source/Clients/NostalgicPlayer/MainWindow/MainWindowForm.cs

```diff
@@ -145,9 +145,6 @@ public MainWindowForm()
 
 			randomList = new List<int>();
 
-			openAgentSettings = new Dictionary<Guid, AgentSettingsWindowForm>();

```

Comment:
I didn't mean those, please put those back :-)

I mean the event assignment below on the searchPopupControl.